src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -932,7 +933,8 @@ UniValue AsyncRPCOperation_sendmany::perform_joinsplit(AsyncJoinSplitInfo & info
     uint256 anchor;
     {
         LOCK(cs_main);
-        pwalletMain->GetNoteWitnesses(outPoints, witnesses, anchor);
+        pwalletMain->GetSproutNoteWitnesses(outPoints, witnesses, anchor);
+        // TODO: Sapling witnesses
```

Comment:
Sapling witnesses won't be fetched here, as Sapling spends are handled in a very different way to Sprout JoinSplits.